src/Microsoft.Health.Fhir.Api/Registration/FhirServerApplicationBuilderExtensions.cs

```diff
@@ -23,10 +27,56 @@ public static IApplicationBuilder UseFhirServer(this IApplicationBuilder app)
 
             app.UseHealthChecksExtension(new PathString(KnownRoutes.HealthCheck));
 
+            var config = app.ApplicationServices.GetService(typeof(IOptions<FhirServerConfiguration>)) as IOptions<FhirServerConfiguration>;
+            EnsureArg.IsNotNull(config, nameof(config));
+
+            var pathBase = new PathString(config.Value.PathBase.TrimEnd('/'));
```

Comment:
https://github.com/microsoft/fhir-server/pull/1200#issuecomment-674832683
> 
> 
> This pull request **introduces 1 alert** when merging [d904924](https://github.com/microsoft/fhir-server/commit/d9049245a5398e2e95b2450629c6434f7ea9d0f0) into [dfbb34c](https://github.com/microsoft/fhir-server/commit/dfbb34cea30bf47abdc13c175be590a37d2307c9) - [view on LGTM.com](https://lgtm.com/projects/g/microsoft/fhir-server/rev/pr-f7329449dc8d1349a6fb1a2b3f6fbec415189d7b)
> 
> **new alerts:**
> 
>     * 1 for Dereferenced variable may be null

It is a false positive (`L33`). There is a null check in `L31`